docs-src/concepts.md

```diff
@@ -106,7 +106,12 @@ configuration. For example, two application teams may inadvertently submit
 configuration for the same HTTP path. There are several different strategies for
 handling this:
 
-* TODO
+* A controller may detect this situation during reconciliation and choose to honor
+  one resource or other, perhaps based on a timestamp (e.g. first write wins).
```

Comment:
Do Kubernetes objects always come with modification timestamps?
I know that creation time is standard.

Reply:
They don't have a modification timestamp at all. They is resourceVersion and generation but there is no way to figure out which was the last updated one.

In our controller, whenever such issues arise, the controller respects the resource which has the oldest creation time.